apps/service-portal/src/hooks/useNavigation/useNavigation.ts

```diff
@@ -23,12 +23,6 @@ const filterNavigationTree = (
           route.path.includes(item.path)),
     ) !== undefined || item.systemRoute === true
 
-  // Only display the document-provider module for those who have access
-  if (item.path === '/skjalaveitur') {
```

Comment:
This should be visible locally, if we really want to only show specific users we can do that in configcat later.